.changeset/popular-roses-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
```

Comment:
Should we release this? I think it is better for it to have a release entry than for it to be released under other work.

Reply:
I don't see why not! There's definitely value in being able to use a rich text control (or multiple) even if the panels appear in the palette in a non-optimal way (i.e., no sub-selection yet). Alan thinks it's fine to release it like this as well while we work on sub-selection, etc.

Reply:
Cool cool